src/routes/(console)/apply-credit/+page.svelte

```diff
@@ -189,6 +189,18 @@
     $: selectedOrg = $organizationList?.teams?.find(
         (team) => team.$id === selectedOrgId
     ) as Organization;
+
+    function getNewBillingPlan(organization: Organization): BillingPlan {
+        if (organization?.billingPlan === BillingPlan.SCALE) {
+            return BillingPlan.SCALE;
```

Comment:
I'm still a bit hesitant to hardcode stuff here. For example, enterprise plans will run into a problem with this logic.

Maybe we can rely on the plan order (after it gets fixed for the education plan)?

Reply:
Yeah definitely! However atm we don't have a plan order in code afaik?

Reply:
@ernstmul We don't have an exact order [there's `tier-*` format though] but possibly we could check via `getNextTier`, not 100% sure as doesn't include anything about enterprise/custom tiers atm.

Reply:
`tier-*` doesn't hold up anymore, since we now also have auto-1 and cont-1! `getNextTier` would definitely work, but perhaps its nice get this info from BE?